commandline/host_vars/puhti-login13.csc.fi

```diff
@@ -0,0 +1,8 @@
+# compile in fast directory
+compile_root: "/local_scratch/{{ ansible_user_id }}"
+is_admin: false
+python3_module: python-data/3.10-23.07
+python3_base_version: 3.10
+python_modcall: ". /appl/profile/zz-csc-env.sh && module load {{python3_module}} &&"
+python_swig_modcall: "{{python_modcall}} module load gcc/11.3.0 && module load swig/3.0.12 &&"
+gcc_modcall: ". /appl/profile/zz-csc-env.sh && module load gcc/11.3.0 &&"
```

Comment:
We could likely avoid having identical variables for puhti-login12.csc.fi and puhti-login13.csc.fi if we used group vars for the hpc group. The variables seem to be defined in the localhost inventory, so those should automatically override the group vars.

Reply:
Fully agree, this bugged me too, but I did not think of the fix quick enough. Will fix.